RunningOrder/Story/Views/StoryList.swift

```diff
@@ -43,12 +42,13 @@ extension StoryList {
                     }
                 }
             }
-            .navigationTitle("Sprint \(sprint.number) - \(sprint.name)")
+            .navigationTitle(logic.isSearchFound ? "Searching" : "Sprint \(sprint.number) - \(sprint.name)")
```

Comment:
why `logic.isSearchFound` instead of `searchManager.isItemSelected` ?

Reply:
yeah ^^' i'll rename